OWNERS

```diff
@@ -1,7 +1,8 @@
 approvers:
 - munnerz
-- kragniz
```

Comment:
:'(

Reply:
Maybe we can add you as an `emeritus_approvers`? https://github.com/kubernetes/kubernetes/blob/2afb403fa1a0b7f3b916bf6fad35aeb1b9a22d25/staging/src/k8s.io/cluster-bootstrap/token/OWNERS#L5-L6

I figure we're best to keep OWNERS files generally up to date to avoid people assigning things to you 😅

Reply:
nah, just removing is fine

Reply:
I'll let you know when I'm around in Bristol next! 😄